docs/usage/modules/versioning.md

```diff
@@ -22,7 +40,36 @@ Configuring or overriding the default `versioning` can be helpful for ecosystems
 }
 ```
 
-The above will override Renovate's default of `docker` versioning for the `python` Docker image and instead use `pep440` versioning to evaluate versions.
+### Linuxserver.io helm-values versioning
+
+If you're using [linuxserver.io](https://www.linuxserver.io/) and want to use their versioning, do the following:
+
+```json
+{
+  "packagePatterns": ["(^|/)linuxserver\\/ddclient$"],
+  "packageNames": ["ddclient"],
+  "versioning": "loose",
+  "allowedVersions": "/^(?<version>v\\d+\\.\\d+\\.\\d+-ls\\d+)$/"
+}
+```
+
+ALTERNATIVE:
+
+If you're using [linuxserver.io](https://www.linuxserver.io/) and want to use their versioning, you can use the config below.
```

Comment:
We don't want to include such specific information here at the top of the versioning modules page, otherwise the expectation will be "why didn't you include my example too?" and we'd end up with pages of example JSON. It's enough that people (a) are alerted to the fact they may need  to explicitly configure versioning, (b) get some brief examples of doing it, (c) create a Discussion if they want further help

Reply:
Oh okay, I was thinking maybe you did want to cover this specific use-case.
It's gone in the next commit. 😄